CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ All user visible changes to this project will be documented in this file. This p
 
 [Diff](/../../compare/3aa35d5bf8ba9728f54db7bf4e21425711097cda...v0.1.0-alpha.6) | [Milestone](/../../milestone/1)
 
+### Added
+
+- Store:
+    - User subscription ([#7], [#4]).
```

Comment:
That is not user-understandable Changelog entry. You should describe changes affecting end users, not what have been added to the code (we have Git log for that).